### PR TITLE
Add "status", "change controller" and "contact" to IANA registration

### PR DESCRIPTION
**Pull request by RyanTheOptimist**
As per feedback from Francesca Palombini

### CODE REVIEW

**Comment by RyanTheOptimist** [comment]
@mnot Would you mind taking a look at this? I think I did the needful, but I'm not sure if there's a better way to do it as I'm not terribly familiar with IANA registrations.

**Comment by mnot** [comment]
Probably better to use a list -- e.g.,

~~~
This document registers a new setting in the "HTTP/3 Settings" registry ({{Section 11.2.2 of HTTP3}}).

* Setting Name: SETTINGS_ENABLE_CONNECT_PROTOCOL
* Default: 0
* Value: 0x08
* Status: permanent
* Date: [ date of publication ]
* Change Controller: HTTP Working Group
* Contact: ietf-http-wg@w3.org
* Notes: 
~~~

**Comment by martinthomson** [comment]
That, but with a definition list:

```
Setting Name:
: SETTINGS_ENABLE_CONNECT_PROTOCOL
Default:
: 0
...
```
```

**Comment by RyanTheOptimist** [comment]
> 

Ah! Thanks guys. PTAL.

File: draft-ietf-httpbis-h3-websockets.md [hunk 1]
```diff
@@ -86,16 +86,25 @@ discussed in {{!RFC8441}}.
...
+: permanent
+Specification:
+: This Document
+Date:
+: [ date of publication ]
+Change Controller:
+: HTTP Working Group
+Contact:
+: ietf-http-wg@w3.org
+Notes:
```
Comment (martinthomson) [inline draft-ietf-httpbis-h3-websockets.md:107]:
```suggestion
Notes:
: <!-- -->
```

Empty items turn out to be tricky.

**Comment by RyanTheOptimist** [comment]
@martinthomson Does this look right to you?

**Review by martinthomson — APPROVED** [review]
Looks like this (in text):

```
   This document registers a new setting in the "HTTP/3 Settings"
   registry (Section 11.2.2 of [HTTP3]).

   Value:  0x08

   Setting Name:  SETTINGS_ENABLE_CONNECT_PROTOCOL

   Default:  0

   Status:  permanent

   Specification:  This Document

   Date:  [ date of publication ]

   Change Controller:  IETF

   Contact:  HTTP Working Group (ietf-http-wg@w3.org)

   Notes:
```

In other words, perfect!

(In case you are interested in getting this, the CI build generates artifacts: hit "Show all checks" and get the "Details" on the CI check, then go to the "Summary" and download the "artifact", which is a zip file containing text and HTML versions of drafts.  Sometimes that is faster than checking out the code and building it yourself...)

**Comment by RyanTheOptimist** [comment]
> (In case you are interested in getting this, the CI build generates artifacts: hit "Show all checks" and get the "Details" on the CI check, then go to the "Summary" and download the "artifact", which is a zip file containing text and HTML versions of drafts. Sometimes that is faster than checking out the code and building it yourself...)

AH! I knew I was missing something!

**Pull request merged**